pinot-plugins/pinot-batch-ingestion/v0_deprecated/pinot-hadoop/src/main/java/org/apache/pinot/hadoop/io/PinotOutputFormat.java

```diff
@@ -171,12 +160,10 @@ private void initSegmentConfig(JobContext context)
       throws IOException {
     _segmentConfig.setFormat(FileFormat.JSON);
```

Comment:
I was wondering why you removed ReaderConfig, then saw that file format is hardcoded to JSON.
Not related to your PR, but shouldn't  `_segmentConfig.setFormat(FileFormat.JSON);` be done in the JSONPinotOutputFormat implementation?

Reply:
This class has multiple things missing. Will clean them up in the following PRs. Next step is to clean up all the empty constructors.